app/models/user.rb

```diff
@@ -18,12 +18,11 @@ class User < ActiveRecord::Base
   enum role: %w(default admin)
 
   def self.find_or_create_by_auth(auth_data)
-
-    user = User.where(id: auth_data['uid'][1..3]).first_or_create
+    user = User.find_or_create_by(id: auth_data['uid'][1..3])
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.